core/state_test.go

```diff
@@ -380,15 +405,6 @@ func TestClass(t *testing.T) {
 	cairo1Class, err := gw.Class(context.Background(), cairo0Hash)
 	require.NoError(t, err)
 
-	err = encoder.RegisterType(reflect.TypeOf(cairo0Class))
-	if err != nil {
-		require.Contains(t, err.Error(), "already exists in TagSet")
-	}
-	err = encoder.RegisterType(reflect.TypeOf(cairo1Hash))
```

Comment:
This was registering a felt with the encoder right? Should we not include it in the set of types we register in TestMain?

Reply:
I'm not sure why it was registered in the first place, because I was trying to mimic the registration behavior in `blockchain/encoder_initializer.go`, which doesn't register the `felt` type.